config.yml

```diff
@@ -0,0 +1,175 @@
+remote:
```

Comment:
I wonder if there is a way to keep the config from the main repo? Imagining this will get a bit tedious to maintain separate config file in each repo and we should try to keep one config in centralised location

Reply:
There are different ways to do this. Please see [this](https://stackoverflow.com/questions/15844542/git-symlink-reference-to-a-file-in-an-external-repository/27770463#27770463) answer. I'm going to highlight two of these ways here.

One option is to create a symbolic link in GAMBLR.viz (and in any other child repo) that points to GAMBLR's `config.yml`. However, there are two details that don't make it so great as it looks:
* Fisrt, it creates a GAMBLR directory (or submodule) in GAMBLR.viz that stores all GAMBLR's files. This submodule is like a separate repo (from GAMBLR.viz), that contains separate commits, just like the GAMBLR repo. Therefore we need space to store the submodel content, which makes GAMBLR.viz bigger. However, the GAMBLR submodel (62M) in GAMBLR.viz is smaller than the GAMBLR repo (242M)!
* Second, if the `config.yml` file is changed in the GAMBLR repo, to update the symbolic link `config.yml` in GAMBLR.viz, we need to manually run `git pull` in the submodule GAMBLR in GAMBLR.viz. At least I couldn't find a way to update it automatically.

Another option is to simply add (`git add`/`commit`) a `fetch_gamblr_config_file.sh` file to GAMBLR.viz to fetch GAMBLR's `config.yml` file and create/update GAMBLR.viz's `config.yml` file. The content of `fetch_gamblr_config_file.sh` would be:
```
#!/bin/sh
curl -O https://raw.githubusercontent.com/morinlab/GAMBLR/master/config.yml
```
But again, I don't know how to automatically trigger the execution of this `.sh` file.

Reply:
There are different ways to do this. Please see [this](https://stackoverflow.com/questions/15844542/git-symlink-reference-to-a-file-in-an-external-repository/27770463#27770463) answer. I'm going to highlight two of these ways here.
One option is to create a symbolic link in GAMBLR.viz (and in any other child repo) that points to GAMBLR's `config.yml`. However, there are two details that don't make it so great as it looks:
* Fisrt, it creates a GAMBLR directory (or submodule) in GAMBLR.viz that stores all GAMBLR's files. This submodule is like a separate repo (from GAMBLR.viz), that contains separate commits, just like the GAMBLR repo. Therefore we need space to store the submodel content, which makes GAMBLR.viz bigger. However, the GAMBLR submodel (62M) in GAMBLR.viz is smaller than the GAMBLR repo (242M)!
* Second, if the `config.yml` file is changed in the GAMBLR repo, to update the symbolic link `config.yml` in GAMBLR.viz, we need to manually run `git pull` in the submodule GAMBLR in GAMBLR.viz. At least I couldn't find a way to update it automatically.
Another option is to simply add (`git add`/`commit`) a `fetch_gamblr_config_file.sh` file to GAMBLR.viz to fetch GAMBLR's `config.yml` file and create/update GAMBLR.viz's `config.yml` file. The content of `fetch_gamblr_config_file.sh` would be:
```
#!/bin/sh
curl -O https://raw.githubusercontent.com/morinlab/GAMBLR/master/config.yml
```
But again, I don't know how to automatically trigger the execution of this `.sh` file.

Reply:
I think it will run at install, yes - and will download the file. It might get out of sync if there are other changes later but it probably won't matter much because without re-installing package the user won't have access to the new functionality anyways. In any case, let's first find out how much do we even need config here anyways and then see how we can handle it

Reply:
I added "config.yml" to GAMBL.viz because `ashm_multi_rainbow_plot` function uses `config::get("results_tables")` internally.

Reply:
Does it use it to get ssm? I think we need to drop this functionality and rely on user providing their own maf data frame

Reply:
What about we keep the `config.yml` file only in GAMBLR repo and in the readme of all GAMBLR child repos we just instruct the user to download `config.yml` from GAMBLR and `setwd()` to where the file was saved.

Reply:
I started [this](https://morinlabsfu.slack.com/archives/C0224H120CU/p1690393930395609) thread on Slack, just making sure you saw this - I think we came up with very similar solution.

Reply:
This was solved by just deleting the config.yml file from all GAMBLR child repos.